src/libraries/Microsoft.Bcl.Cryptography/src/System/Security/Cryptography/IncrementalHash.netfx.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    internal sealed class IncrementalHash : IDisposable
```

Comment:
We already have a polyfill for this over at https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/System/Security/Cryptography/IncrementalHash.netfx.cs

Delete this one and compile in the existing one?